cs/configuring.texy

```diff
@@ -292,16 +292,55 @@ Lze konfigurovat některé parametry [Tracy |tracy:] a nastavovat panely do Debu
 
 /--neon
 	tracy:
-		email: %webmasterEmail%
+		# email, na který se posílají notifikace, že došlo k chybě
+		email: dev@example.com
+		fromEmail: robot@example.com
+
+		# ve vývojovém režimu zobrazí chyby typu notice nebo warning jako BlueScreen
 		strictMode: true
-		editor: ...
+		
+		# zobrazí umlčené (@) chybové hlášky
+		scream: true
+
+		# formát odkazu pro otevření v editoru
+		editor: editor://open/?file=%file&line=%line
+
+		# cesta k otevření prohlížeče s uloženým BlueScreen
```

Comment:
cesta k prohlížeči, který bude automaticky otevírat zalogované BlueScreen v CLI režimu (nejsem si jist, ale tuším to funguje jen v CLI)

Reply:
Vypada to, ze to funguje jenom v CLI, ano.

https://api.nette.org/2.4/source-Tracy.Debugger.php.html#327